cmd/node/node.go

```diff
@@ -657,6 +665,58 @@ func (app *SpacemeshApp) startServices() {
 	go app.checkTimeDrifts()
 }
 
+func (app *SpacemeshApp) startAPIServices(postClient api.PostAPI, net api.NetworkAPI) {
+	apiConf := &app.Config.API
+
+	// OLD API SERVICES (deprecated)
```

Comment:
remove?

Reply:
The plan is not to remove the old API until a later phase, once everything is implemented. For now I've marked it as "deprecated" in a few places. To be clear, some things like the community block explorer and the spacemesh app are relying on the old API today and still need to be updated. Let's discuss elsewhere.